src/main/java/htsjdk/samtools/CRAMBAIIndexer.java

```diff
@@ -125,14 +124,12 @@ public void processContainer(final Container container, final ValidationStringen
 
         int sliceIndex = 0;
         for (final Slice slice : container.slices) {
-            slice.containerOffset = container.offset;

```

Comment:
this is now set elsewhere

Reply:
`Container.setSlices()` sets this value.  If the slices exist prior to the setting of this value, then `Container.setByteOffset()` sets the slices' values.